docs/local_setup_sans_docker.md

```diff
@@ -1,4 +1,4 @@
-# Developing Klaxon Locally
+# Setting up Klaxon locally for development
```

Comment:
Just for clarity, wanted to differentiate the docs for this branch and the sans Docker setup